src/CMakeLists.txt

```diff
@@ -73,6 +73,9 @@ add_library(slicot SHARED
 
 target_link_libraries(slicot lapack_aux ${ExtLibs})
 
+set_property(TARGET slicot PROPERTY VERSION "0.0.0")
+set_property(TARGET slicot PROPERTY SOVERSION 0 )
+
 install(TARGETS slicot
-        LIBRARY DESTINATION lib
-        ARCHIVE DESTINATION lib)
+        LIBRARY DESTINATION ${LIB_INSTALL_DIR}
```

Comment:
I searched for it. It seems that Fedora has a cmake macro for that. But all I can find is with setting a variable per hand. Either on Fedora or Debian (or others). --- I was very happy that you made a cmake build of it and I got it to work. Meson looks great.